gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs/components/api-runtime-logs-list/api-runtime-logs-list.component.html

```diff
@@ -68,12 +68,21 @@
       <td mat-cell *matCellDef="let log">{{ log.gatewayResponseTime + 'ms' }}</td>
     </ng-container>
 
+    <ng-container matColumnDef="endpoint">
+      <th mat-header-cell *matHeaderCellDef id="endpoint">Endpoint reached</th>
+      <td mat-cell *matCellDef="let log">
+        @if (log.endpoint != null) {
```

Comment:
```suggestion
        @if (log.endpoint) {
```
In case it is `undefined` and not `null`?

Reply:
done